### PR TITLE
feat: new layout algorithm

### PR DESCRIPTION
**Pull request by marstamm**
Replaces the existing layouting algorithm with another one that is more resilient against loops.

Adds bundeling for esm and cjs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +28,10 @@
     "bpmn",
     "layout"
   ],
-  "author": "hoferch91",
```
Comment (philippfromme) [inline package.json:23]: 👋🏻

Comment (marstamm) [reply package.json:23]: Don't want to annoy them with support cases ;)

**Review by philippfromme — APPROVED** [review]
Let's gooo! 🚀

**Comment by corocoto** [comment]
@marstamm, great job 👏  Thanks a lot for the PR!

**Pull request merged**